Rocket.Chat/Managers/Launcher/AnalyticsCoordinator.swift

```diff
@@ -44,9 +44,14 @@ struct AnalyticsCoordinator: LauncherProtocol {
     }
 
     private func launchFirebase() {
-        #if RELEASE || BETA
```

Comment:
@rafaelks wouldn't be better to leave it here to let it clear that we should only track events on these environments?